Patches lfhex to compile fine with -DNDEBUG (which is typically used to
compile release versions without debugging information).

The original version made an essential function call within an assert()
statement - but defining NDEBUG makes any assert statements into no-ops.

Patch written 2008 by Guenther Brunthaler <gb_about_gnu@gmx.net>
Index: src/reader.cpp
===================================================================
--- src.orig/reader.cpp
+++ src/reader.cpp
@@ -278,7 +278,11 @@ uchar Reader::operator[] (off_t offset)
 		       "attempt to access past end of file");
 
   off_t page_idx = offset/_pageSize;
+  #ifdef NDEBUG
+  (void)loadPage( page_idx );
+  #else
   assert( loadPage( page_idx ) );
+  #endif
   return _data[page_idx][ offset%_pageSize ];
 }
 
